var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -45,14 +45,19 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     # URL must remain http:// so Spack can bootstrap curl
     url = "http://www.cpan.org/src/5.0/perl-5.24.1.tar.gz"
 
-    # Development releases
+    # see http://www.cpan.org/src/README.html for
+    # explanation of version numbering scheme
+
+    # Development releases (odd numbers)
     version('5.25.11', '37a398682c36cd85992b34b5c1c25dc1')
 
-    # Maintenance releases (recommended)
-    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c', preferred=True)
-    version('5.22.3', 'aa4f236dc2fc6f88b871436b8d0fda95')
+    # Maintenance releases (even numbers, recommended)
+    version('5.26.2', 'dc0fea097f3992a8cd53f8ac0810d523', preferred=True)
+    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c')
 
     # Misc releases that people need
+    version('5.22.4', '31a71821682e02378fcdadeed85688b8')
```

Comment:
I would add it to "End of life releases". Honestly, maybe we should move all of these "Misc releases" to "End of life releases".

Reply:
done.